ConnectorGrasshopper/ConnectorGrasshopperShared/Accounts/Accounts.GetAccountToken.cs

```diff
@@ -46,12 +47,7 @@ public Accounts_GetAccountToken()
 
   protected override void RegisterInputParams(GH_InputParamManager pManager)
   {
-    pManager.AddTextParameter(
-      "Account",
-      "A",
-      "Account to get the auth token from. Expects the `userId`",
-      GH_ParamAccess.item
-    );
+    pManager.AddParameter(new SpeckleAccountParam());
```

Comment:
This changes the input parameter type, which would be a breaking change for users with GH scripts with this component that are passing in a text parameter. Are we okay with this @AlanRynne ?

Reply:
Nevermind, looks like the input node was an `Accounts` node which previously passed the user id as the output now passes the `Account` object. Looks good to go ✌️